platform/util/platform-util-unavailableurls/src/main/java/org/codice/ddf/platform/util/http/UnavailableUrls.java

```diff
@@ -116,17 +120,31 @@ public boolean contains(String url) {
         }
 
         /**
-         * Computes the next timeout, doesn't exceed MAX_TIMEOUT_SECONDS
+         * Computes the next timeout, doesn't exceed maximum internval
          */
         private void backoff() {
-            timeout = Math.min(timeout * INITIAL_TIMEOUT_SECONDS, MAX_TIMEOUT_SECONDS);
+            if (timeout >= getMaxRetryInterval() / 2) {
+                timeout = getMaxRetryInterval();
+            } else {
+                timeout += timeout;
+            }
```

Comment:
Change the algo a bit. The previous implementation multiplied the last timeout interval by 10 on each retry. This implementation doubles the interval every time.